clang/test/CIR/CodeGen/complex.init-list.c

```diff
@@ -0,0 +1,44 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -Wno-unused-value -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --input-file=%t.cir %s -check-prefix=CIR
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -Wno-unused-value -fclangir -emit-llvm %s -o %t-cir.ll
+// RUN: FileCheck --input-file=%t-cir.ll %s -check-prefix=LLVM
```

Comment:
Renaming the file to `complex-init-list.c` would probably be best. The number of tests cases seems small enough that perhaps you can merge them altogether in `./CodeGen/complex.c`?

Reply:
Oh gotcha, that's fine!